src/components/TutorialController.jsx

```diff
@@ -0,0 +1,50 @@
+import * as CONSTANTS from "../constants/constants";
+import Button from "./Button";
+
+const TutorialController = ({
+  orderButton,
+  isComplatedIndex,
+  isComplatedButton,
+  setRotationAngle,
+  setIsComplatedButton,
+}) => {
+  const handleClickLeft = () => {
+    if (!isComplatedButton) {
+      setRotationAngle((prevAngle) => prevAngle - 90 * CONSTANTS.DEGREE);
+      setIsComplatedButton(true);
+    }
+  };
+
+  const handleClickRight = () => {
+    if (!isComplatedButton) {
+      setRotationAngle((prevAngle) => prevAngle + 90 * CONSTANTS.DEGREE);
+      setIsComplatedButton(true);
+    }
+  };
+
+  return (
+    <div className="flex flex-col gap-4 fixed bottom-4 right-4 w-[250px] rounded-md bg-slate-600 p-5 justify-center items-center">
+      <div className="w-[90%] flex gap-2 items-center">
+        <p className="pl-[2%] pr-[15%] text-center font-bold text-lg">Turn!</p>
+        <Button
+          className="h-10 p-1"
+          clickHandler={handleClickLeft}
+          disabled={
+            (!isComplatedIndex || orderButton === "right") && "disabled"
```

Comment:
```suggestion
            (!isComplatedIndex || clickRequestButton === "right") && "disabled"
```

orderButton도 의미가 맞는 뜻이나 위처럼 변경한다면 좀 더 직관적으로 의미가 다가올 것 같습니다!

Reply:
코드 리뷰 내용 반영했습니다! 감사합니다!